.github/workflows/issue-manager.yml

```
@@ -2,20 +2,28 @@ name: "Issue Manager"
 
 on:
   schedule:
-  - cron: "0 0 * * *"
+    - cron: "0 0 * * *"
+  issue_comment:
+    types:
+      - "created"
+  issues:
+    types:
+      - "labeled"
+  pull_request_target:
+    types:
+      - "labeled"
 
 jobs:
   issue-manager:
-    runs-on: ubuntu-latest
+    runs-on: "ubuntu-latest"
     steps:
-    - uses: tiangolo/issue-manager@master
+    - uses: "tiangolo/issue-manager@master"
```

```suggestion
    - uses: "tiangolo/issue-manager@0.4.0"
```
It's probably worth pinning to the latest release and using dependabot to keep the action up to date by adding `github-actions`  to it like:
https://github.com/ASFHyP3/hyp3-sdk/blob/develop/.github/dependabot.yml

---

Agree, good catch.